docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.3"
+
+services:
+  gateway:
+    image: "[your gateway image]"
+    container_name: gateway
+    hostname: gatewayhost
+    networks:
+      - sianet
+    ports:
+      - 50000:80
```

Comment:
The format `HOST:CONTAINER` may cause YAML parsing issues. Consider using double quotes to enforce parsing as string.

See: https://docs.docker.com/compose/compose-file/#ports